sdk/src/contexts/mpc.rs

```diff
@@ -1,12 +1,11 @@
-use crate::network::NetworkMultiplexer;
-use round_based::PartyIndex;
 use std::collections::BTreeMap;
-use std::sync::Arc;
+use subxt_core::tx::signer::Signer;
 use subxt_core::utils::AccountId32;
+use crate::contexts::{ServicesContext, TangleClientContext};
 
 /// `MPCContext` trait provides access to MPC (Multi-Party Computation) functionality from the context.
 #[async_trait::async_trait]
-pub trait MPCContext {
+pub trait MPCContext: ServicesContext<Config = <Self as TangleClientContext>::Config> + TangleClientContext {
```

Comment:
Can you create a MPCClient and just have this context return this client? This way we can debug the impl a lot more easily?